go.mod

```diff
@@ -26,6 +27,7 @@ require (
 	k8s.io/client-go v0.20.1
 	k8s.io/code-generator v0.20.1
 	k8s.io/component-base v0.20.1
+	k8s.io/klog/v2 v2.4.0
 	k8s.io/kube-aggregator v0.20.1
 	k8s.io/kube-openapi v0.0.0-20201113171705-d219536bb9fd
 	k8s.io/kubernetes v1.20.1
```

Comment:
for what did we need this dependency in the past?

Reply:
k8s.io/kubernetes? I am not sure :/

Reply:
https://github.com/openshift/oauth-apiserver/pull/9#discussion_r574518257